src/Package/Impl/Publishing/Handlers/RmdPublishHandler.cs

```diff
@@ -25,12 +29,45 @@ internal sealed class RmdPublishHandler : IMarkdownFlavorPublishHandler {
             return true;
         }
 
-        public string GetCommandLine(string inputFile, string outputFilePath, PublishFormat publishFormat, Encoding encoding) {
-            string format = GetDocumentTypeString(publishFormat);
+        public async Task Publish(IRSession session, string inputFilePath, string outputFilePath, PublishFormat publishFormat, Encoding encoding) {
+            // Send the markdown to R-Host for processing as a blob
+            byte[] rmdData = File.ReadAllBytes(inputFilePath);
+            var rmdBlobId = await session.SendBlobAsync(rmdData);
```

Comment:
I'd make extension to the session for sending files so it would look like session.SendFileAsync() and  hide reading bytes